html/pfappserver/root/config/switch/list.tt

```diff
@@ -73,23 +78,26 @@
         <tbody>
           [% FOREACH switch IN items %]
           <tr>
-            <td><a href="[% c.uri_for(c.controller('Config::Switch').action_for('view'), [ switch.id ]) %]">[% switch.id | html %]</a>
+            <td><a href="[% c.uri_for(c.controller(controller_namespace).action_for('view'), [ switch.id ]) %]">[% switch.id | html %]</a>
```

Comment:
c.controller(controller_namespace).action_for 

can be rewritten as

c.controller.action_for

Since c.controller() will return the current controller when there are no arguments passed to it.